acra-censor/common/file_log_storage.go

```diff
@@ -7,7 +7,6 @@ import (
 
 // FileLogStorage is a file-based implementation of LogStorage interface
 type FileLogStorage struct {
-	filePath       string
 	file           *os.File
 	openedToAppend bool
```

Comment:
do we need it more?

Reply:
no it seems we didn't. I used it when reopening file with other modes set, but now Truncate allows to avoid it